package.json

```diff
@@ -2,17 +2,21 @@
   "name": "cord-field",
   "version": "3.0.0-alpha",
   "scripts": {
-    "start": "react-app-rewired start",
+    "start": "PORT=3001 react-app-rewired start",
     "build": "react-app-rewired build",
     "test": "react-app-rewired test",
     "lint": "eslint --ext .ts,.tsx --fix --max-warnings 0 .",
     "type-check": "tsc -p .",
-    "analyze": "yarn build --analyze"
+    "analyze": "yarn build --analyze",
+    "generate": "graphql-codegen --config codegen.js",
+    "generate:watch": "graphql-codegen --watch --config codegen.js"
```

Comment:
Is there not a default config location?

Reply:
@CarsonF the default config is `codegen.yaml` - we want to use JS so that we can do things like read environment variables